### DISCUSSION TITLE
Unable to overwrite .npmrc in renovate/renovate:full

### DISCUSSION THREAD

**Discussion by purnhar** [renovatebot/renovate#30397] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Self configured GitLab-Runner, Renovate Version 38.4.0

### Please tell us more about your question or problem

We start renovate as GitLab job, copied from the gitlab-runner project. In the pre-build-script of our GitLab-Runner configuration we overwrite several files in the home-directory of the container to setup configurations and caches.

When we try to write `~/.npmrc` we get a `Permission denied` error, because the underlying `/opt/containerbase/home/.npmrc` is not writeable for UID 1000. It is necessary for us to modify this file to set up npm mirrors. Could you switch the ownership this file to the ubuntu-user?

### Logs (if relevant)

<details><summary>Logs</summary>

```

cp: cannot create regular file '/home/ubuntu/.npmrc': Permission denied

```

</details>

**Comment by rarkins** [comment]
Yes, we should fix that

**Reply by viceice** [reply]
https://github.com/containerbase/base/commit/6a3f01d54de46a93556c448d68fe3cd6efb90acf

**Reply by viceice** [reply, marked as answer]
- https://github.com/renovatebot/renovate/pull/30407

this should fix it
